src/workspacer.Tests/workspacer.Tests.csproj

```diff
@@ -15,9 +11,13 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
   </ItemGroup>
+  <ItemGroup>
```

Comment:
The changes to this project file seems to be non-significant and should be reverted to create a smaller diff/create fewer merge-conflicts.

Reply:
Fixed in 070577a7f7b93f7a422950b420a6cec842cded18